### PR TITLE
Fix typo

### LINKED ISSUE #1703: Spell checks / Typos
I use a VS Code extension: [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)

Going through the exercises I've found a few instances where there are some obvious typos etc.

This issue will probably be excessively pedantic, but I think if anyone else has a few spelling fixes, maybe they can add them to this issue for tracking?

I'll start it off:

[exercises/practice/doubly-linked-list/src/pre_implemented.rs#L1C19-L1C23](https://github.com/exercism/rust/blob/dd0ea4e6c165ebe67d8d2257333cc9bd4e9c6a4d/exercises/practice/doubly-linked-list/src/pre_implemented.rs#L1C19-L1C23) - `thes` -> `this`

### PR DESCRIPTION
**Pull request by senekor**
[no important files changed]

close #1703

### CODE REVIEW

**Comment by senekor** [comment]
@exercism/reviewers should be an easy review :-)

**Comment by senekor** [comment]
Thank you! I added it more as a note to myself, to not forget. And I think PR descriptions got into the suggested body of the commit.

Weird, I still can't merge :thinking:

**Comment by senekor** [comment]
Alright thanks! I'll be waiting on @ErikSchierboom's review then. I think I'm the only active maintainer, unless I should ping @dem4ron to review my PRs?

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**
